code/controllers/subsystem/timer.dm

```diff
@@ -422,24 +438,21 @@ SUBSYSTEM_DEF(timer)
  * Removes this timed event from any relevant buckets, or the secondary queue
  */
 /datum/timedevent/proc/bucketEject()
```

Comment:
you need to clear `bucket_joined ` in here so that if the timer is joined back in (such as looping timers with a wait higher then can fit in the buckets) it doesn't bitch about the bucket already being joined.

Reply:
bucket_joined is a just debug flag by the way